### PR TITLE
Heroku Deployment

### LINKED ISSUE #16: Hosting the Node App on Heroku/AWS
## Description
As a contributor,
I need ensure that my Node App is live for others to use.

### Update [Required]
- [ ] server.js
Will update the other files once I am able to fix the issue.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
20 hours

@ancyp @Janiceilene Can you please assign me the issue to work upon?

### PR DESCRIPTION
**Pull request by BaaniLeen**
### Description
Deployed the NodeApp on Heroku

Fixes #16 

### Type of Change:

- Code
- Quality Assurance
- User Interface
- Outreach
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Ran the tests locally.

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: auth/github.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ passport.deserializeUser(function (id, fn) {
 passport.use(new GitHubStrategy({
     clientID: "94c3bea3e111456764eb",
     clientSecret: "b665c678fe607b3460b50c41fb01d801848f207a",
-    callbackURL: "http://127.0.0.1:3000/auth/github/callback"
+    callbackURL: "https://intense-plateau-42820.herokuapp.com/auth/github/callback"
```
Comment (bethanyr) [inline auth/github.js]: this should be the systers heroku url or ideally based on an environment variable.

Comment (BaaniLeen) [reply auth/github.js]: Sure. :)
I need admin access to create a heroku app, as soon as that is done, I will update this :)

**Comment by BaaniLeen** [comment]
@bethanyr @ancyp Updated the new URL for the slack-systers-open-source
 app here :)

**Comment by BaaniLeen** [comment]
@bethanyr @ancyp Updated the link to the new heroku app! :)

**Pull request merged**